<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Photos</class>
 <widget class="QWidget" name="Photos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>272</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>272</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>272</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Photos</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>458</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="expand_pushButton">
      <property name="minimumSize">
       <size>
        <width>35</width>
        <height>30</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>放大</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="narrow_pushButton">
      <property name="minimumSize">
       <size>
        <width>35</width>
        <height>30</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>缩小</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="previous_pushButton">
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>30</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>上一张</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="next_pushButton">
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>30</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>下一张</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="open_pushButton">
      <property name="minimumSize">
       <size>
        <width>35</width>
        <height>30</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>浏览</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="delete_pushButton">
      <property name="minimumSize">
       <size>
        <width>35</width>
        <height>30</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>删除</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="adjust_pushButton">
      <property name="minimumSize">
       <size>
        <width>35</width>
        <height>30</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>退出</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>120</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>请点击浏览按钮！</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
